com.unity.render-pipelines.universal/Shaders/Utils/StencilDeferred.shader

```diff
@@ -387,6 +413,7 @@ Shader "Hidden/Universal Render Pipeline/StencilDeferred"
             #pragma multi_compile_fragment _ _DEFERRED_MIXED_LIGHTING
             #pragma multi_compile_fragment _ _SCREEN_SPACE_OCCLUSION
             #pragma multi_compile_fragment _ _LIGHT_LAYERS
+            #pragma multi_compile_fragment _ _ADDITIONAL_LIGHT_COOKIES
```

Comment:
It is dangerous for deferred light shaders to use the same shader keyword than forward shaders (`_ADDTIONAL_LIGHT_COOKIES`). As each deferred light can toggle the keyword on/off, when you get to forward-only or transparent pass, the original setting for the keyword as set up by ForwardLights may have been set to an incorrect value.

I would suggest renaming to `_DEFERRED_ADDITIONAL_LIGHT_COOKIES` in order to preserve the original setting for `_ADDITIONAL_LIGHT_COOKIES`.

Reply:
This is a good point. I will rename it.
Also I'll merge the Main and Additional light cookies defines to reduce variants on forward side.
I think the renaming is still worth doing as it's kind of deferred special thing.

Reply:
Fixed.